tests/tas-elasticsearch/src/test/resources/test-suites/elasticsearch-suite.xml

```diff
@@ -1,13 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
-<suite name="REST API tests Elasticsearch" preserve-order="true">
-  <listeners>
-    <listener class-name="org.alfresco.utility.report.HtmlReportListener"/>
-    <listener class-name="org.alfresco.utility.testrail.TestRailExecutorListener"/>
-    <listener class-name="org.alfresco.utility.testng.OSTestMethodSelector"/>
-  </listeners>
-
-  <test name="elasticsearch">
+<suite name="REST API tests Elasticsearch" preserve-order="true" verbose="1">
+  <test name="elasticsearch" verbose="3" preserve-order="true">
```

Comment:
What does verbose="1" and "3" mean?  Are they meant to be set to different things from each other?